jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -274,17 +275,32 @@ class PPSCreateHandler(BaseHandler):
 
     @tornado.web.authenticated
     async def get(self, path):
-        """Get the pipeline spec for the specified notebook."""
-        response = await self.pps_client.generate(path)
-        await self.finish(response)
+        try:
+            """Get the pipeline spec for the specified notebook."""
```

Comment:
The docstring should be the first line of the function

Reply:
Oops, thanks! Done.